gulp/tasks/copy.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var changed = require( 'gulp-changed' );
+var replace = require( 'gulp-replace' );
```

Comment:
L:4 and L:5 technically should be `gulpChanged` and `gulpReplace`, but perhaps that gets too wordy in the implementation.